editor/gui/scene_tree_editor.cpp

```diff
@@ -1336,7 +1333,21 @@ bool SceneTreeEditor::can_drop_data_fw(const Point2 &p_point, const Variant &p_d
 		}
 	}
 
-	return String(d["type"]) == "nodes" && filter.is_empty();
+	if (String(d["type"]) == "nodes" && filter.is_empty()) {
```

Comment:
```suggestion
	if (filter.is_empty() && String(d["type"]) == "nodes") {
```
Cheaper condition first